librz/core/cmd_descs/cmd_descs.yaml

```diff
@@ -254,9 +254,8 @@ commands:
     summary: Seek commands
     subcommands: cmd_seek
   - name: t
-    cname: cmd_type
```

Comment:
I think you should keep both `cname` and `type` here if you want the oldinput `t` handler to keep working.